java-checks/src/main/java/org/sonar/java/checks/tests/JUnit5SilentlyIgnoreClassAndMethodCheck.java

```diff
@@ -40,8 +45,14 @@ protected void raiseIssueOnNonCompliantReturnType(MethodTree methodTree) {
     Type type = returnType.symbolType();
     boolean methodReturnAValue = !type.isUnknown() && !type.isVoid();
     if(methodReturnAValue && !methodTree.symbol().metadata().isAnnotatedWith("org.junit.jupiter.api.TestFactory")) {
-      reportIssue(returnType, "Replace the return type by void.");
+      QuickFixHelper.newIssue(context)
+        .forRule(this)
+        .onTree(methodTree.returnType())
+        .withMessage("Replace the return type by void.")
+        .withQuickFixes(() -> List.of(JavaQuickFix.newQuickFix("Replace with void")
```

Comment:
You can use `.withQuickFix` (singular) instead of `.withQuickFixes` and pass it the single `JavaQuickFix` created, instead of instantiating a `List` 

```suggestion
     .withQuickFix(() -> 
        JavaQuickFix.newQuickFix("Remove modifier")
        .addTextEdit(JavaTextEdit.removeTree(modifier))
        .build())